### PR TITLE
chore: update pictograms to be fluid in their containers

### PR DESCRIPTION
**Pull request by vpicone**
Updates icon size to 96px and uses fewer columns to accommodate.

I brought up the idea of a size dropdown for future work on this page to @laurenmrice. This just provides a default size for now.

### CODE REVIEW

**Comment by mjabbink** [comment]
the download card should live in usage not library

**Comment by vpicone** [comment]
@mjabbink I was thinking about having a download icon on each card, did you mean for the master ai file?

**Comment by mjabbink** [comment]
@vpicone @laurenmrice The usage page gallery needs attention. There seems to be a margin applied to several images.

Screen Shot 2019-09-30 at 3 09 59 PM



Screen Shot 2019-09-30 at 3 10 10 PM

**Comment by vpicone** [comment]
@mjabbink might want to open a seperate issue for that since it's broken in idl and carbon either spot works.

**Comment by mjabbink** [comment]
@vpicone @laurenmrice 

- [ ] Mobile needs larger pictograms.

File: src/components/SvgLibraries/PictogramLibrary/PictogramCategory.js [hunk 1]
```diff
@@ -9,14 +9,20 @@ import {
...
     <h2 className={cx(h2, categoryTitle)}>{category}</h2>
-    <ul className={svgGrid}>
+    <ul className={cx(svgGrid, pictogramList)}>
       {pictograms.map(pictogram => (
-        <SvgCard key={pictogram.name} icon={pictogram} />
+        <SvgCard
+          key={pictogram.name}
+          icon={pictogram}
+          height="35%"
+          width="35%"
```
Comment (vpicone) [inline src/components/SvgLibraries/PictogramLibrary/PictogramCategory.js]:
```suggestion
          Test
```

**Pull request merged**